ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/ReferenceTestProtocolSchedules.java

```diff
@@ -73,9 +73,8 @@ public static ReferenceTestProtocolSchedules create() {
     builder.put(
         "Merge",
         createSchedule(new StubGenesisConfigOptions().mergeNetSplitBlock(0).baseFeePerGas(0x0a)));
-    builder.put(
-        "Shanghai",
-        createSchedule(new StubGenesisConfigOptions().shandongBlock(0).baseFeePerGas(0x0a)));
+    builder.put("Shanghai", createSchedule(new StubGenesisConfigOptions().shanghaiTimestamp(0)));
```

Comment:
Should be able to use Shandong instead of Shanghai still. Shandong is being removed anyway. 
Once this PR is in, we can start migrating Shandong guards -> Shanghai